debugging_protected_audience_api_services.md

```diff
@@ -132,6 +132,12 @@ PS_VERBOSITY = "10"
 The value of `CONSENTED_DEBUG_TOKEN` should match the debug token set in the
 device. The value must be a non-empty string.
 
+Require consented token min length 6.
+Adtech use cases:
+1. set the token to empty string -> consent log is turned off
+2. set the token not empty but less than min length -> server will crash
+3. set the token to at least min length -> consent log is turned on
+
```

Comment:
I can't find the change proposed by Akshay:

```
The value of CONSENTED_DEBUG_TOKEN should match the debug token set in the
device. If set, the value must be a string of length 6 or more characters.

The behavior of CONSENTED_DEBUG_TOKEN is as following:

Setting to an empty string: This will turn consented debugging off for that server.
Setting to a string of length 6 or more characters: This causes consented debugging to be turned on, with the given value as the debug token for that server.
Setting to a non-empty string less than 6 characters will prevent the server from starting.
```